src/main/java/uk/sky/cqlmigrate/LockConfig.java

```diff
@@ -26,6 +28,10 @@ String getClientId() {
         return clientId;
     }
 
+    public boolean getUnlockOnFailure() {
```

Comment:
agree, had doubts though readability vs conventions. thanks for backing readability! 😄